### DISCUSSION TITLE
¿Cómo hacer componentes independientes? VUEX

### DISCUSSION THREAD

**Discussion by RaulRomero26** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#477] [Q&A] [answered]
Hola, tras realizar la explicación del día jueves, tengo la duda de como poder realizar un mismo componente, pero que sea independiente, por ejemplo: 
![image](https://user-images.githubusercontent.com/55547165/159305033-8908b3df-8787-4e67-922d-0533ca0b57fe.png)
 Si yo realizo esto voy a tener dos componentes contador, sin embargo ambos se "suman o restan" a la vez, eso es lo quiero saber, ¿como hacerlos independientes?

![image](https://user-images.githubusercontent.com/55547165/159305343-852439d7-3752-49d6-8309-48941ae2beea.png)

**Comment by chistoperez** [comment, marked as answer]
Necesitarías tener otro estado contador para que guarde el del segundo contador, ahí están apuntando al mismo los dos.
También pasarle como props el contador al componente para que sepa a cual contador va a hacerle la chamba.
`<Contador contador="1"  />`
`<Contador contador="2"  />`
eso de contador="1" es la propiedad que le vas a pasar al contador para que sepa cual va a sumar o restar, y adentro del componente tienes que recibir ese valor
https://codingpotions.com/vue-props
checa esta pagina, ahi explican como pasar y recibir props.
Suerte

**Reply by RaulRomero26** [reply]
si funciono después de mucho logre por lo menos que imprimiera cosas diferentes, gracias

**Comment by doguedogue** [comment]
Exacto, como dice @chistoperez al pasarle una propiedad al componente va a guardar el estado, y con eso van a ser independientes ambos componentes del contador.

Te paso un repo de ejemplo de un vato que da cursos en Udemy, es muy bueno, y está precisamente ese ejemplo, agrega varios componentes Counter en App.vue para que veas lo que preguntas.
https://github.com/Klerith/vue-fundamentos-cli/tree/fin-seccion-4

**Reply by RaulRomero26** [reply]
Gracias, igual tomo cursos de este profesor, y aun asi sabiendo Angular se me esta complicando la cosa con vue.
